### ISSUE TITLE
[Feature][state] seatunnel 如何实现自定义状态

### ISSUE THREAD

**Issue by Xuzhengz** [apache/seatunnel#6359]
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/seatunnel/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

seatunnel 如何实现自定义状态

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by Carl-Zhou-CN** [comment]
hi，@Xuzhengz   In what context does he need to apply it

**Comment by oomgomgxx** [comment]
> hi，@Xuzhengz In what context does he need to apply it

譬如用作周期性sql增量查询的依据。
参考logstash：https://www.elastic.co/guide/en/logstash/current/plugins-inputs-jdbc.html#_predefined_parameters

seatunnel能实现同样的功能？

**Comment by liunaijie** [comment]
> > hi，[@Xuzhengz](https://github.com/Xuzhengz) In what context does he need to apply it
> 
> 譬如用作周期性sql增量查询的依据。 参考logstash：https://www.elastic.co/guide/en/logstash/current/plugins-inputs-jdbc.html#_predefined_parameters
> 
> seatunnel能实现同样的功能？

No, seatunnel is stateless, this value need control or pass by the client side
